tests/packages/configure-data/meson.build

```diff
@@ -13,8 +13,10 @@ py = py_mod.find_installation()
 configure_file(
     input: 'configure_data.py.in',
     output: 'configure_data.py',
-    configuration: configuration_data({
-        'MSG': 'hello!',
-    }),
+    configuration: configuration_data(
+        {
+            'MSG': 'hello!',
+        },
+    ),
     install_dir: py.get_install_dir(),
```

Comment:
This was forced because we had a trailing comma there. The trick is to not have a trailing comma at all, so that it inlines the dictionary. black works the same way.

I think the reasoning is that when you are gonna add another argument to the function, it will have to change the style significantly by moving the dictionary brackets, so in these cases you simply remove the trailing comma to allow it to  shrink the dictionary definition.